library/ecp.c

```diff
@@ -1464,7 +1464,10 @@ static int ecp_add_mixed( const mbedtls_ecp_group *grp, mbedtls_ecp_point *R,
     return( MBEDTLS_ERR_ECP_FEATURE_UNAVAILABLE );
 #else
     int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
-    mbedtls_mpi T1, T2, T3, T4, X, Y, Z;
+    mbedtls_mpi T1, T2, T3, T4;
+    mbedtls_mpi * const X = &R->X;
```

Comment:
Minor: I'm slightly concerned that the aliases might hide the fact that we're writing to the output, hence potentially overwriting the input. Perhaps leaving `&R->X` and so on in the code would make that more apparent? Either ways, we should leave a comment about it.

Reply:
Agree